magellan-legacy/src/main/java/com/wealthfront/magellan/Navigator.kt

```diff
@@ -99,14 +109,28 @@ public class Navigator internal constructor(
 
   public fun isCurrentScreen(other: NavigableCompat): Boolean = currentNavigableProvider!!.isCurrentNavigable(other)
 
-  public fun currentScreen(): NavigableCompat = currentNavigableProvider!!.navigable!!
+  public fun currentScreen(): NavigableCompat? = currentNavigableProvider!!.navigable
```

Comment:
Made `currentScreen` nullable to reflect the magellan 2.0 change and also added a couple of utility method for making the migration easier